unwrap_test.go

```diff
@@ -53,3 +53,76 @@ func TestUnwrapFrame_badPC(t *testing.T) {
 		t.Errorf("inner: got %v, want %v", inner, giveErr)
 	}
 }
+
+// caused follows the pkg/errors Cause interface
+type caused struct {
+	err error
+}
+
+func (e *caused) Error() string {
+	return e.err.Error()
+}
+
+func (e *caused) Cause() error {
+	return e.err
+}
+
+func TestUnwrapOnce(t *testing.T) {
```

Comment:
I generally recommend testing against the exported interface, so that the internal implementation can change without impacting tests.

Reply:
Sure. This test was both a legacy of prior review changes from when UnwrapOnce was exported and an attempt to get to 100% code coverage, which I normally am not overly concerned with (80% seems fine to me).